ksp_plugin/vessel.cpp

```diff
@@ -645,6 +648,44 @@ void Vessel::AttachPrediction(
   }
 }
 
+bool Vessel::IsCollapsible() const {
+  PileUp* containing_pile_up = nullptr;
+  std::set<not_null<Part*>> parts;
+  for (const auto& [_, part] : parts_) {
+    // Not collapsible if any part has a force or a torque applied to it.
+    if (part->intrinsic_force() != Vector<Force, Barycentric>{}) {
+      return false;
+    }
+    if (part->intrinsic_torque() != Bivector<Torque, Barycentric>{}) {
+      return false;
+    }
+    // Not collapsible if any part is not piled up (not even sure when that can
+    // happen).
```

Comment:
Maybe we should try to understand what is going in our own code, instead of treating it like KSP?
This should either be removed or be a CHECK; not being piled up is something transient when building pile ups, and we should definitely not be considering collapsibility when the pile ups are being determined.